content(news): Add some new news messages (#10274)
endless-sky/endless-sky: data/human/news.txt

			"The other day I happened across a rock band whose transport flaked out. Good thing I had seats for all eight of them, because it got me into a free concert at their next stop."
			"Always look for work that provides a lot of detail. The more interesting it sounds, the more likely it is to end up giving you stories to tell your kids someday."
			"For some reason, boring jobs almost never hang out in the spaceport. So, if you are looking for interesting or at least varied work..."
			"If you can't stand the heat, stay out of the solar storms."
			"Silence is golden, except when you are listening to the life support systems."

> 			`Even in this day and age some people still say "the sun goes down", instead of the more appropriate "the sun goes behind the horizon".`

Reviewer: ```suggestion
			`Even in this day and age some people still say "the sun goes down," instead of the more appropriate "the sun goes behind the horizon."`
```

> 			`Even in this day and age some people still say "the sun goes down," instead of the more appropriate "the sun goes behind the horizon."`

Author: ```suggestion
			"Even in this day and age some people still say 'the sun goes down,' instead of the more appropriate 'the sun goes behind the horizon.'"
```

			"For some reason, boring jobs almost never hang out in the spaceport. So, if you are looking for interesting or at least varied work..."
			"If you can't stand the heat, stay out of the solar storms."
			"Silence is golden, except when you are listening to the life support systems."

> 			`Even in this day and age some people still say "the sun goes down," instead of the more appropriate "the sun goes behind the horizon."`

Reviewer 2: ```suggestion
			"Even in this day and age some people still say 'the sun goes down,' instead of the more appropriate "the sun goes behind the horizon.'"
```

Author: Fixed, including the "-character before the sun.

			"Some Flat Earthers accused me of playing propaganda videos on the ship's viewports when they looked out of the ship while we were in orbit."
			"There's no point crying over spilt milk. But the bill for cleaning the 200 tons of milk that leaked from my ship is no laughing matter either."
			"I spy with my little eye... something large, flat, and sturdy. The landing pads! How did you know?"
			"A Starbarge, by any other name, is still just a moving cargo container."
			"I named my ship Titanic. It's a strong sounding name, but I feel people are making fun of me."

> 			"The early Firebird gets the Wormhole."

Author: This doesn't really feel like a spoiler to me (humanity knows about the theory of Wormholes since Albert Einstein wrote about them in 1915), but this news-item does work better in a Hai context, so I moved it to `"hai news.txt"`.

			"A single spray of this nanotech perfume will make you irresistible to any prospective partners! Just don't use it when close to a dog. Or any other animal."
			"Can I offer you a lucky playing card deck? You're guaranteed to get perfect hands in every game you play! What? No! It's not just a pile of 52 aces!"
			"This laser razor never needs sharpening! Achieve a smoking hot look every time!"
			"All that glitters is not gold, but it still is valuable as long as we can sell it to some fool."

> 			`'Something old, something new, something borrowed, something blue,' we sell it all at our nice shop near you.`

Reviewer 2: ```suggestion
			"'Something old, something new, something borrowed, something blue,' we sell it all at our nice shop near you."
```
The backticks are not necessary anymore.

> 			"'Something old, something new, something borrowed, something blue' we sell it all at our nice shop near to you."

Reviewer: ```suggestion
			`"Something old, something new, something borrowed, something blue", we sell it all at our nice shop near you.`
```

			"Can I offer you a lucky playing card deck? You're guaranteed to get perfect hands in every game you play! What? No! It's not just a pile of 52 aces!"
			"This laser razor never needs sharpening! Achieve a smoking hot look every time!"
			"All that glitters is not gold, but it still is valuable as long as we can sell it to some fool."

> 			`"Something old, something new, something borrowed, something blue," we sell it all at our nice shop near you.`

Reviewer 2: ```suggestion
			`'Something old, something new, something borrowed, something blue,' we sell it all at our nice shop near you.`
```

			"A trouble shared is a trouble halved, so most resource issues can be brought within acceptable limits by just pretending to listen."
			"A man is judged by his deeds, not by his words, except in my profession; the more I talk, the more people value me."
			"People are sometimes afraid that we psychologists can read their minds, but we are actually more scared of learning our own motivations during our studies."
			"Better to be rejected for who you are, than to be accepted for whom you are not."
			"Leave behind what you don't need, to create space to find what makes you progress."
			"Better comforted by a lie, than hurt by the truth. Hurt people are really unproductive."

> 			"Difficult times creates strong persons. But we are here for when it gets a little too difficult."

Reviewer: ```suggestion
			"Difficult times create strong people. But we are here for when it gets a little too difficult."
```

news "investigator"
	location
		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
		attributes "rich" "urban"
	name
		word
			"Detective"
			"Investigator"
	message
		word
			`"`
		word

> 			"Better to have loved and lost, so that you can hire me to find back."

Author: Better to have loved and lost, than to never had a good reason at all to hire me to find back?

> 			"Better to have loved and lost, so that you can hire me to find back."

Reviewer: Find _it_ back, maybe. Yes, sounds good.

			"I will work for food."
			"I will work for transport out of this place."
			"I barely have any money today, but that will change; just wait until my ship comes in."
			"Don't trust anyone over 30. They are the ones that got us in this mess in the first place."

> 			"People have been complaining about the `Kids these days` since 624 BC. Wonder how humanity survived this long."

Reviewer: ```suggestion
			`People have been complaining about the "Kids these days" since 624 BC. Wonder how humanity survived this long.`
```

> 			`People have been complaining about the 'kids these days' since 624 BC. Wonder how humanity survived this long.`

Reviewer 2: ```suggestion
			"People have been complaining about the 'kids these days' since 624 BC. Wonder how humanity survived this long."
```

> 			"OK Boomer. Nice of you to visit us, now be on your way again."

Reviewer: ```suggestion
			"OK, boomer. Nice of you to visit us, now be on your way again."
```

			"I will work for food."
			"I will work for transport out of this place."
			"I barely have any money today, but that will change; just wait until my ship comes in."
			"Don't trust anyone over thirty. They are the ones that got us in this mess in the first place."

> 			`People have been complaining about the "kids these days" since 624 BC. Wonder how humanity survived this long.`

Reviewer 2: ```suggestion
			`People have been complaining about the 'kids these days' since 624 BC. Wonder how humanity survived this long.`
```

			"All's fine and dandy until someone dumps a ton of rotten Caldan lobsters and you have to deal with the smell."
			"My worth comes from removing what's worthless!"
			"You wouldn't believe some of the stuff people throw away around the spaceport. Yesterday I found a perfectly intact ${small trash item} that someone had dumped."

> 			"My shuttle got serious damage when some ${small trash item} exploded because it was not properly disposed."

Reviewer: ```suggestion
			"My shuttle got serious damage when some ${small trash item} exploded because it was not properly disposed of."
```

			"If rich kids are so pampered that they've never so much as had to wash a dish in their entire lives, it makes me wonder how they hope to be independent adults one day."
			"The kids here are not spoiled at all; the kids in the family I work for typically don't get a new computer game until two days after it gets on the market."
			"I just got fired. The dog of one of the kids died and the missus asked me to fly to Earth and buy a new identical one. When I returned she was screaming that she didn't need two dead dogs."
			"There's one law for the rich and another law for the poor. One small mistake by me and I'm back to the horrible place I emigrated from."

> 			`I don't know what part of 'no' those spoiled kids didn't understand, but I'm not cooking 5 course dinners every day when they eat almost nothing.`

Reviewer 2: ```suggestion
			"I don't know what part of 'no' those spoiled kids didn't understand, but I'm not cooking 5 course dinners every day when they eat almost nothing."
```

			"If rich kids are so pampered that they've never so much as had to wash a dish in their entire lives, it makes me wonder how they hope to be independent adults one day."
			"The kids here are not spoiled at all; the kids in the family I work for typically don't get a new computer game until two days after it gets on the market."
			"I just got fired. The dog of one of the kids died and the missus asked me to fly to Earth and buy a new identical one. When I returned she was screaming that she didn't need two dead dogs."
			"There's one law for the rich and another law for the poor. One small mistake by me and I'm back to the horrible place I emigrated from."

> 			`I don't know what part of "no" those spoiled kids didn't understand, but I'm not cooking 5 course dinners every day when they eat almost nothing.`

Reviewer 2: ```suggestion
			`I don't know what part of 'no' those spoiled kids didn't understand, but I'm not cooking 5 course dinners every day when they eat almost nothing.`
```

Reviewer: Why?

> news "painter"

Reviewer 3: This probably needs a `location`.

	name
		word
			"Painter"
	message
		word

> 			"He who can, creates; he who cannot, transports. Sorry, no offence meant to you."

Reviewer 2: ```suggestion
			"He who can, creates; he who cannot, transports. Sorry, no offense meant to you."
```

news "painter"
	location
		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
		attributes "tourism"
	name
		word
			"Painter"
	message
		word

> 			"He who can, creates; he who cannot, transports. Sorry, no offense meant to you."

Reviewer 2: This entry probably also needs quotes around it.
```suggestion
			`"He who can, creates; he who cannot, transports. Sorry, no offense meant to you."`
```

			"Antigrav boards for rent! You break it, you pay."
			"I sometimes misread 'wine tasting' as 'time wasting.' The worst part is that nobody's noticed so far."
			"Looking for some company for tonight? No questions asked, no strings attached. Just contact me if you need anything."
			"Burn the candle at both ends. Sign up for our romantic zero gravity dinner!"

> 			"Friends, fellow planetman, and travellers. Welcome to today's tour of this wonderful spaceport."

Reviewer: ```suggestion
			"Friends, fellow planetmen, and travellers. Welcome to today's tour of this wonderful spaceport."
```

			"Like new, low mileage, previously owned by an elderly lady."
			"Hey, you there! I have the perfect ship for you."
			"One of my ships has a fuel leak? No way, it's... sweat from all the hard work it does."

> 			"Don't ask where I got my merchandise. Those shuttles fell out off the cargo-hold of a transport ship."

Reviewer 2: ```suggestion
			"Don't ask where I got my merchandise. Those shuttles fell out of the cargo hold of a transport ship."
```

			"Of course I sell junk. If the ships were any good then the previous owners would have kept them."
			"Help feed the children. My children. Come buy a ship."
			"Those Firebirds are not old junk! Those are classics!"
			"Every ship I sell has a free extra vehicular suit and free repair kit included. But you will rarely need it."

> 			"How much credits do you have to spend on a new ship? I have something in every price-range."

Reviewer 2: ```suggestion
			"How many credits do you have to spend on a new ship? I have something in every price range."
```

> 			"Why buy that fuel-in-efficient Flivver, when you can save 100 credits per year on fuel by spending 150k credits more on a Heavy Shuttle?"

Reviewer 2: ```suggestion
			"Why buy that fuel inefficient Flivver, when you can save 100 credits per year on fuel by spending 150k credits more on a Heavy Shuttle?"
```

			"We hired a marketing agency and they came up with the sloppy slogan 'Just what's needed to get the job done.' Can you believe we spent huge amounts of money on that garbage?"
			"An employee once asked me how I was able to buy such a luxurious Flivver. I said that if she keeps working hard, soon I'll be able to buy another one."
			"A colleague of mine is in deep trouble: he was tired of having to use his badge every time to enter the office, so he just disabled all the security. It's going to take him years to repay all the equipment that was stolen."
			"You think you can work here? Don't call us, we'll call you."
			"Does this corporation have a communications problem? Why didn't I hear about that?"
			"Last year we were standing on the edge, but we made huge steps forward."
			"People that work make mistakes. I know people that never make any mistakes."
			"You don't need to be crazy to work here, but it does help quite a bit."
			"If you look for mistakes, then use a mirror and not binoculars."

> 			"If you are up to your neck in the shit, then don't let you head down."

Reviewer: Let's not use bad words here. 
```suggestion
```

		word
			"A startup company once hired me, and I immediately advised them to stop; hiring an expensive consultant like me is a horrible decision for a cash strapped startup."
			"Working as the head of a company requires sound decision making abilities. Hiring an expensive consultant like me usually suggests a lack thereof, but I'm not complaining since I get paid well."

> 			"My first advice is always free. And let me just give it to you; I strongly recommend hiring me."
> 			"Here is a piece of free advice; I strongly recommend hiring me."

Reviewer 2: ```suggestion
			"My first advice is always free. And let me just give it to you: I strongly recommend hiring me."
			"Here is a piece of free advice: I strongly recommend hiring me."
```

		word
			"A startup company once hired me, and I immediately advised them to stop; hiring an expensive consultant like me is a horrible decision for a cash strapped startup."
			"Working as the head of a company requires sound decision making abilities. Hiring an expensive consultant like me usually suggests a lack thereof, but I'm not complaining since I get paid well."

> 			"My first advice is always free. And let me just give it to you: I strongly recommend hiring me."

Reviewer 4: "advice" -> "piece of advice"

			"We've got student interns in the lab this month. A few of them are very bright!"
			"Excuse me, captain, I have to resume work on a paper I'm co-authoring. Would you like an advance copy?"
			"My research concerns force fields. No, not those kinds of force fields."
			"I'm working on a top secret project; even I don't know what I'm doing."

> 			"Working on a top secret project; would love to tell you, but then I need to kill you. Haha just kidding, I'm doing research on producing staples more cheaply."

Reviewer 3: ```suggestion
			"Working on a top secret project; would love to tell you, but then I'd need to kill you. Haha just kidding, I'm doing research on producing staples more cheaply."
```

			"Can I speak to the manager? There was a scratch on my cabin-door and it really irritated me. I think my whole vacation is ruined now."
			"Can I speak to the manager? The landing on this planet was a bit rough and I'm seriously considering filing a complaint."
			"Warn me if you see my partner or my secret lover. They learned about each other and now they're both trying to kill me."
			"Get away from me! I already told twenty of your colleagues that I don't need a taxi service."

> 			"You there, where are my bags! What do you mean by you don't know? You look like one of those dockworkers."

Author: Resolved with different quotes (since the double quotes were already in use around this text).

> 			"We are not amused. The Starqueen Victoria is delayed by 2 minutes, and I need to wait here among the commoners with no privacy whatsoever."

Author: https://en.wikipedia.org/wiki/Royal_we

> 			"We are not amused. The Starqueen Victoria is delayed by 2 minutes, and I need to wait here among the commoners with no privacy whatsoever."

Reviewer: Right. I wasn't aware it was being used this way.
```suggestion
			"We are not amused. The Star Queen Victoria is delayed by two minutes, and I need to wait here among the commoners, with no privacy whatsoever."
```

			"Can I speak to the manager? There was a scratch on my cabin-door and it really irritated me. I think my whole vacation is ruined now."
			"Can I speak to the manager? The landing on this planet was a bit rough and I'm seriously considering filing a complaint."
			"Warn me if you see my partner or my secret lover. They learned about each other and now they're both trying to kill me."

> 			"Get away from me! I already told 20 of your colleagues that I don't need a taxi service."

Reviewer: ```suggestion
			"Get away from me! I already told twenty of your colleagues that I don't need a taxi service."
```

			"Can I speak to the manager? There was a scratch on my cabin-door and it really irritated me. I think my whole vacation is ruined now."
			"Can I speak to the manager? The landing on this planet was a bit rough and I'm seriously considering filing a complaint."
			"Warn me if you see my partner or my secret lover. They learned about each other and now they're both trying to kill me."
			"Get away from me! I already told twenty of your colleagues that I don't need a taxi service."

> 			"You there, where are my bags! What do you mean by 'you don't know'? You look like one of those dockworkers."

Reviewer 4: Can this be changed to using double quotes for 'you don't know'?

Author: No, because there are already double-quotes around this whole section. (It would look strange to use double quotes here too.)

			"Can I speak to the manager? There was a scratch on my cabin-door and it really irritated me. I think my whole vacation is ruined now."
			"Can I speak to the manager? The landing on this planet was a bit rough and I'm seriously considering filing a complaint."
			"Warn me if you see my partner or my secret lover. They learned about each other and now they're both trying to kill me."
			"Get away from me! I already told twenty of your colleagues that I don't need a taxi service."

> 			"You there, where are my bags! What do you mean by 'you don't know'? You look like one of those dockworkers."

Reviewer: ```suggestion
			"You there, where are my bags! What do you mean by 'you don't know?' You look like one of those dockworkers."
```

> 			"We are not amused. The Star Queen Victoria is delayed by two minutes, and I need to wait here among the commoners, with no privacy whatsoever."

Reviewer: ```suggestion
			"We are not amused. The Star Queen Victoria is delayed by two minutes, and we need to wait here among the commoners, with no privacy whatsoever."
```
Maybe?

Author: I feel this works better with `I` in the second part, because otherwise this person could be talking about a group of people, instead of the Royal_We prefix.

Reviewer: True.

			"Nobody thinks about the Alphas anymore, but don't forget that they just disappeared. They could return at any time. If that happens, I'm going to kill every last one of 'em."
			"While it seems pointless to have the Bactrian locked behind a license, don't forget that it's one of the oldest designs in human space. It was the Navy Carrier of its time - and it's still comparable to one."
			`We recently went to full alert, because somebody screamed "It's scrambled!" Turns out some rookie in the galley was enthusiastic about making scrambled eggs.`
			"Curiosity killed the cat. The R.N.S. Curiosity landed right on top of the poor creature."
			"We arrested a fortune teller yesterday; they didn't see it coming."
		word
			`"`

news "republic soldier [bragging]"
	location
		government "Republic"
		attributes "military"
	name
		word
			"Republic soldier"
	message
		word
			`"`
		word
			"The last words of that "
			"The famous last words of that "
		word
			"pirate"
			"rebel"
			"spy"
			"troublemaker"
			"terrorist"
		word

> 			" were; "

Reviewer 2: ```suggestion
			" were: "
```

		word
			"Don't you dare shoot that gun!"
			"I am faster than you!"
			"I'm making a lot of money with this scheme. You want to be partners?"
			"I can make you rich, if you join me."

> 			"My father will be very angry if you kill me."

Author: Resolved on line 1713 and after the section.

> 			"My friends will come after you if you shoot me."

Author: Resolved on line 1713 and after the section.

> 			"Oh no!"

Author: Resolved on line 1713 and after the section.

> 			"Please don't kill me!"

Author: Resolved on line 1713 and after the section.

> 			"Run, the Navy is here!"

Author: Resolved on line 1713 and after the section.

> 			"You don't have jurisdiction here."

Author: Resolved on line 1713 and after the section.

> 			"You think you can hit me over that distance?"

Reviewer 2: ```suggestion
			"You think you can hit me over that distance?"
			"What are you gonna do? Shoot me?"
```

Author: Nice addition.

> 			"You think you can hit me over that distance?"

Author: Resolved on line 1713 and after the section.

			`We recently went to full alert, because somebody screamed "It's scrambled!" Turns out some rookie in the galley was enthusiastic about making scrambled eggs.`
			"Curiosity killed the cat. The R.N.S. Curiosity landed right on top of the poor creature."
			"We arrested a fortune teller yesterday; they didn't see it coming."
		word
			`"`

news "republic soldier [bragging]"
	location
		government "Republic"
		attributes "military"
	name
		word
			"Republic soldier"
	message
		word
			`"`
		word
			"The last words of that "
			"The famous last words of that "
		word
			"pirate"
			"rebel"
			"spy"
			"troublemaker"
			"terrorist"
		word
			" were; "
		word
			"Don't you dare shoot that gun!"
			"I am faster than you!"
			"I'm making a lot of money with this scheme. You want to be partners?"
			"I can make you rich, if you join me."
			"My father will be very angry if you kill me."
			"My friends will come after you if you shoot me."
			"Oh no!"
			"Please don't kill me!"

> 			"Run, the navy is here!"

Reviewer: ```suggestion
			"Run, the Navy is here!"
```

			`"`

news "convicted criminal"
	location
		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
	name
		word
			"Convict"
			"Convicted Criminal"
			"Transported Convict"
	message
		word
			`"`
		word
			"I fought the law, and the law won."

> 			"I'm innocent, I tell you! They convicted me for 5 murders, but I only killed 2 persons!"

Reviewer: ```suggestion
			"I'm innocent, I tell you! They convicted me for five murders, but I only killed two people!"
```

			"Two years of food and housing for free, and all the time in the world to read. I'm so looking forward to staying in prison."
			"I should have gone to one of those pirate worlds. They would never have caught me."
			"They said I took a bribe to lose the game, but that is not true! I would also have gotten the credits if we had won!"

> 			"The rich just buy themselves out of legal troubles! The Major was in on the plan, but he got away with a fine of 100 credits and I'm here serving 5 years!"

Reviewer: ```suggestion
			"The rich just buy themselves out of legal troubles! The Mayor was in on the plan, but he got away with a fine of 100 credits and I'm here serving 5 years!"
```

			`"`

news "convicted criminal"
	location
		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
	name
		word
			"Convict"
			"Convicted Criminal"
			"Transported Convict"
	message
		word
			`"`
		word
			"I fought the law, and the law won."
			"I'm innocent, I tell you! They convicted me for five murders, yet I only killed two people!"
			"Two years of food and housing for free, and all the time in the world to read. I'm so looking forward to staying in prison."
			"I should have gone to one of those pirate worlds. They would never have caught me."
			"They said I took a bribe to lose the game, but that is not true! I would also have gotten the credits if we had won!"

> 			"The rich just buy themselves out of legal troubles! The Mayor was in on the plan, but he got away with a fine of 100 credits and I'm here serving five years!"

Reviewer 4: 100 credits seems too low, given that a passenger transport is like 5k. Maybe 10000 credits?

Author: I will change it to 1000 credits, the Mayor still got away really easily. :)

			"Living on the edge."
			"You can't win the game if you're afraid to lose it."
			"Good things come to those who wait, and then I rob those from them very quickly."
			"Two heads are better than one, since you can put a spiked head on each side of the road if you have two."
			"It takes two to make a bargain. Just kill the first fool that says no, and the second one will quickly agree to my demands."
			"I fought the law, and I won."

> 			"Sorry, need to take a call; Hi, this is Joe from Syndicate support. You seem to have a problem with your flight computer."

Reviewer 2: ```suggestion
			"Sorry, need to take a call. 'Hi, this is Joe from Syndicate support. You seem to have a problem with your flight computer.'"
```

Reviewer: @tibetiroka don't you mean
```suggestion			`Sorry, need to take a call. "Hi, this is Joe from Syndicate support. You seem to have a problem with your flight computer."`
```
?

Reviewer 2: No, the news entry is already in quotes. There is a line above and below that adds it.

			"Living on the edge."
			"You can't win the game if you're afraid to lose it."
			"Good things come to those who wait, and then I rob those from them very quickly."
			"Two heads are better than one, since you can put a spiked head on each side of the road if you have two."
			"It takes two to make a bargain. Just kill the first fool that says no, and the second one will quickly agree to my demands."
			"I fought the law, and I won."
			"Sorry, need to take a call; Hi, this is Joe from Syndicate support. You seem to have a problem with your flight computer."

> 			"Me a pirate? Come on! Do I look like the type of person that kills 21 colonists on a Corvette in Eltanin? Chops them up? Sells their remains as meat to local restaurants? And then go dine in one of those restaurants?"

Reviewer: ```suggestion
			"Me a pirate? Come on! Do I look like the type of person that kills twenty-one colonists on a Corvette in Eltanin? Chops them up? Sells their remains as meat to local restaurants? And then go dines in one of those restaurants?"
```
Is Eltanin a place that we know about in-game?

> 			"I kill people for money, but you are nice, I'll kill you for free."

Reviewer: ```suggestion
			"I kill people for money, but you are nice, so I'll kill you for free."
```

			"Living on the edge."
			"You can't win the game if you're afraid to lose it."
			"Good things come to those who wait, and then I rob those from them very quickly."
			"Two heads are better than one, since you can put a spiked head on each side of the road if you have two."
			"It takes two to make a bargain. Just kill the first fool that says no, and the second one will quickly agree to my demands."
			"I fought the law, and I won."

> 			"Sorry, need to take a call. 'Hi, this is Joe from Syndicate support. You seem to have a problem with your flight computer.'"

Reviewer 4: Can the single quotes be double quotes here? Sorry, I'm not sure if backticks are supported in this place.

Reviewer: They are.

			"Living on the edge."
			"You can't win the game if you're afraid to lose it."
			"Good things come to those who wait, and then I rob those from them very quickly."
			"Two heads are better than one, since you can put a spiked head on each side of the road if you have two."

> 			"It takes two to make a bargain. Just kill the first fool that says no, and the second one will quickly agree to my demands."

Reviewer: ```suggestion
			"It takes two to make a bargain. Just kill the first fool that says no, and the second one will quickly agree to any demands."
```

Author: I think this works better with "my demands", because we have a pirate talking here about how he/she is doing business, and "my demands" sounds more self-centered (which is what I expect from someone that kills to get his/her demands met).

Reviewer: I suppose, though I'm still not sure if the pirate is _telling_ you.

Author: It also works if the pirate is talking to another pirate. The pirate doing the talking is self-centered, regardless of whom he/she is speaking to.

			"Pirates may look fierce, but the moment you start pushing against most of them, they fold like paper."
			"I came here due to all of those oppressive rules and regulations that the Republic force down your throat. Now I'm dealing with all the oppressive demands and threats that pirates force down my throat. It's a start."
			"Beneath the exterior of many pirates, there's a person who desperately wishes that they'd never taken that path and could return to living normally. I'm talking from personal experience."
			"Abandon all hope ye who enter here."

> 			"A Fool and his money are soon parted, on this world."

Reviewer: ```suggestion
			"A fool and his money are soon parted, on this world."
```

			"A video of fast spaceships racing around a three dimensional course in space is shown on the screen, interwoven with clips of very luxurious ships where people appear to be watching the race."
			"An action-packed advertisement about joining the Navy is playing on the screen. It shows impressive video-fragments of carriers, gunships, dropships and heroic marines."
			"Some video is showing Syndicate equipment. The video ends with the slogan, 'Just what's needed to get the job done.'"
			"Six days cruise to hunt pirates, shoot real pirates for 2000 credits a day. 400 bullets included, for free."

> 			"Buy local, buy Southbound Haulers. Also for merchants not from the South."

Author: I changed it a bit (it now talks about where to buy).

			"A video of fast spaceships racing around a three dimensional course in space is shown on the screen, interwoven with clips of very luxurious ships where people appear to be watching the race."
			"An action-packed advertisement about joining the Navy is playing on the screen. It shows impressive video-fragments of carriers, gunships, dropships and heroic marines."
			"Some video is showing Syndicate equipment. The video ends with the slogan, 'Just what's needed to get the job done.'"

> 			"Six days pirate hunting cruise, shoot real pirates for 2000 credits a day. 400 bullets included for free."

Reviewer: ```suggestion
			"Six days pirate on a hunting cruise, and shoot real pirates for 2000 credits a day. 400 bullets included, for free."
```

Author: I'll rewrite it a bit.

Author: Rewrote it to make it more clear that people go on a cruise to hunt pirates.

Reviewer: That part looks good.

			"Dear spaceship captains: please be aware that our outfitters are open all hours of the day. There is never a reason to delay the purchase of your next piece of Syndicate equipment."
			"Dear traveler: we recommend you to have dinner in our local ${culinary venue} '${restaurant names}'. You will not be disappointed."
			"Attention course participants: The course 'Efficient Passenger Logistics' has been canceled because the trainer got stuck in transit."

> 			"Attention course participants: The course communication for professionals is canceled because the provider miscommunicated the date to the instructor."

Reviewer 3: ```suggestion
			"Attention course participants: The course 'Communication for Professionals' is canceled because the provider miscommunicated the date to the instructor."
```

Reviewer: ```suggestion
			`Attention course participants: The course "Communication for Professionals" is canceled because the provider miscommunicated the date to the instructor.`
```

			"Dear spaceship captains: please be aware that our outfitters are open all hours of the day. There is never a reason to delay the purchase of your next piece of Syndicate equipment."
			"Dear traveler: we recommend you to have dinner in our local ${culinary venue} '${restaurant names}'. You will not be disappointed."
			"Attention course participants: The course 'Efficient Passenger Logistics' has been canceled because the trainer got stuck in transit."

> 			"Attention course participants: The course 'Communication for Professionals' is canceled because the provider miscommunicated the date to the instructor."

Reviewer 5: ```suggestion
			"Attention course participants: The course 'Communication for Professionals' is canceled because the provider failed to provide the date to the instructor."
```

			"KEEP FINGERS CLEAR OF GRINDER"
			"REPORT ALL WORKPLACE INJURIES"
			"PROTECTIVE EQUIPMENT REQUIRED WITHIN HAZARD ZONE"
			"Keep hands on railing when traversing stairs."

> 			"Sensitive electronics. Use Electrostatic Discharge protection in this area."

Reviewer 2: ```suggestion
			"Sensitive electronics. Use electrostatic discharge protection in this area."
```

> 			"Strong magnets in use. Don't enter production hall with metal prothesis."

Reviewer: Prothesis?
```suggestion
			"Strong magnets in use. Don't enter the production hall with metal on your body."
```

Author: Sorry, the word I meant is "prostetics", artifical limbs, artifical tooth (some have metal in them).

Hospitals with MRI scanners typically ask their employees not to carry metals around, but they also ask them if they have artificial metal bones or other metals within their bodies (and then don't allow them near the MRI scanner)

			"KEEP FINGERS CLEAR OF GRINDER"
			"REPORT ALL WORKPLACE INJURIES"
			"PROTECTIVE EQUIPMENT REQUIRED WITHIN HAZARD ZONE"

> 			"Keep hands on railing when traversing stairs."

Reviewer 4: Should these also be all caps given the other ones are?

Author: They are not shown all at the same time. I guess this is a style thing.

			"Veni, vidi, hyperjumped away."
			"Those who dance will be called fools by those who cannot hear the music."
			"Better half a sentence, "
			"To the moon, and not coming back."
			"On a small space-station / early in the morning / seven spaceships landed / nicely in a line / and the captain / transmitted departure / tick tick woosh woosh / here we go"

> 			"Time flies like an Arrow, fruit flies like a Banana."

Reviewer: ```suggestion
			"Time flies like an Arrow, fruit flies like a banana."
```

			"Veni, vidi, hyperjumped away."
			"Those who dance will be called fools by those who cannot hear the music."
			"Better half a sentence, "
			"To the moon, and not coming back."
			"On a small space-station / early in the morning / seven spaceships landed / nicely in a line / and the captain / transmitted departure / tick tick woosh woosh / here we go"
			"Time flies like an Arrow, fruit flies like a Banana."

> 			"The Panda eats shoots and leaves. The hunter eats, shoots and leaves."

Reviewer: ```suggestion
			"The panda eats shoots and leaves. The hunter eats, shoots and leaves."
```

> 			"While the man was hunting the deer ran through the forest."

Reviewer: ```suggestion
			"While the man was hunting the deer it ran through the forest."
```
Unless it's intended not to make as much sense.

Author: I'll remove it. It doesn't work here.

			"Better half a sentence, "
			"To the moon, and not coming back."
			"On a small space-station / early in the morning / seven spaceships landed / nicely in a line / and the captain / transmitted departure / tick tick woosh woosh / here we go"
			"Time flies like an Arrow, fruit flies like a banana."
			"The panda eats shoots and leaves. The hunter eats, shoots and leaves."
			"The hyperlane to hell is paved with good intentions."

> 			"Because I could not stop for Death - He kindly stopped for me - The Shuttle held but just one bunk - Most Peacefully. / We slowly flew; He knew no haste - And I had put away - My labor and my leisure too, - For His Civility." # Based on Emily Dickinsons "Because I could not stop for Death"
> 			"We passed the Carrier, where Soldiers strove - In Fighters; in the Void - We passed the Bays with Docked Drones - We passed the Hardened Hull. / Or rather; It passed Us - The Dew drew quivering and Chill - For decompressed, my spacesuit - My tether, torn away." # Based on Emily Dickinsons "Because I could not stop for Death"
> 			"We paused before a House that seemed - A Cylinder to be launched - The Roof was scarcely visible - The Flag; covering around. / Since then, 'tis Centuries, and yet - Feels shorter than the Day - I first surmised the Torpedo Bay - Aimed toward Eternity." # Based on Emily Dickinsons "Because I could not stop for Death"

Reviewer 2: ```suggestion
			"Because I could not stop for Death - He kindly stopped for me - The Shuttle held but just one bunk - Most Peacefully. / We slowly flew; He knew no haste - And I had put away - My labor and my leisure too, - For His Civility." # Based on Emily Dickinson's "Because I could not stop for Death"
			"We passed the Carrier, where Soldiers strove - In Fighters; in the Void - We passed the Bays with Docked Drones - We passed the Hardened Hull. / Or rather; It passed Us - The Dew drew quivering and Chill - For decompressed, my spacesuit - My tether, torn away." # Based on Emily Dickinson's "Because I could not stop for Death"
			"We paused before a House that seemed - A Cylinder to be launched - The Roof was scarcely visible - The Flag; covering around. / Since then, 'tis Centuries, and yet - Feels shorter than the Day - I first surmised the Torpedo Bay - Aimed toward Eternity." # Based on Emily Dickinson's "Because I could not stop for Death"
```
